gno.land/cmd/gnoland/start_test.go

```diff
@@ -24,24 +22,29 @@ func TestStartInitialize(t *testing.T) {
 
 	for _, tc := range cases {
 		name := strings.Join(tc.args, " ")
-		t.Run(name, func(t *testing.T) {
-			mockOut := bytes.NewBufferString("")
-			mockErr := bytes.NewBufferString("")
-			io := commands.NewTestIO()
-			io.SetOut(commands.WriteNopCloser(mockOut))
-			io.SetErr(commands.WriteNopCloser(mockErr))
-			cmd := newRootCmd(io)
+		in, err := NewMockStdin(name)
```

Comment:
I think this is a bad name, since we’re just trying to bufferize the stdout and not the stdin; we actually don’t care about stdin.

Name should be MockStdoutErr; or should just be MockIOs in general.